mobile/ios/ios-emulator-services.ts

```diff
@@ -208,7 +208,12 @@ class IosEmulatorServices implements Mobile.IiOSSimulatorService {
 				this.$logger.trace("Unable to kill simulator: " + e);
 			}
 
-			this.$childProcess.exec(`xcrun simctl launch ${runningSimulatorId} ${appIdentifier}`).wait();
+			setTimeout(() => {
+				// Killall doesn't always finish immediately, and the subsequent
+				// start fails since the app is already running.
+				// Give it some time to die before we attempt restarting.
+				this.$childProcess.exec(`xcrun simctl launch ${runningSimulatorId} ${appIdentifier}`);
```

Comment:
> You are not .wait()-ing the exec call. Is this intended?

Yes. It throws an error if you wait for it outside the fiber... thing.

> Can we use $childProcess.spawn instead? .exec runs a sub-shell, which probably is not needed in this case.

I'd have to push all parameters in an array and uglify the code even more. I suggest we stick with the shell.